docs/content/querying/timeboundaryquery.md

```diff
@@ -9,6 +9,7 @@ Time boundary queries return the earliest and latest data points of a data set.
     "queryType" : "timeBoundary",
     "dataSource": "sample_datasource",
     "bound"     : < "maxTime" | "minTime" > # optional, defaults to returning both timestamps if not set 
+    "filter"    : { "type": "and", "fields": [<filter>, <filter>, ...] } # optional
```

Comment:
hmm, fair enough, I guess this style predated your edit so feel free to leave it that way.